charts/controller/values.yaml

```diff
@@ -9,12 +9,20 @@ service:
 ingress:
   enabled: False
   host: ''
-  path: '/*'
-  servicePort: 7488
+  path: '/'
+  servicePort: 80
```

Comment:
@gchuev-opsfleet if it's a TLS termination proxy, doesn't it make more sense that it would listen on 443?

Reply:
@lirook oh! It's a mistake(copy/paste -_-), there should be same value as for controller.port, I'll updates default values.  This is port of backend for ingress in case of controller default value is 7488.

Reply:
fixed in last commit